packages/clay-autocomplete/stories/index.tsx

```diff
@@ -52,10 +54,23 @@ const ClayAutocompleteWithState = () => {
 	const loading = networkStatus < 4;
 	const error = networkStatus === 5;
 
+	useArrowKeysForList(
+		activeIndex,
+		resource ? resource.results.length : 0,
+		setActiveIndex,
+		inputRef.current
+	);
+
+	useEnterKey(
+		() => alert(resource.results[activeIndex].name),
+		inputRef.current
```

Comment:
This seems strange, just out of curiosity, why did we pass the reference from the input and not from `<ClayAutocomplete.DropDown />`, had problems with this?